crates/uv-cli/src/lib.rs

```diff
@@ -2509,6 +2513,9 @@ pub struct RemoveArgs {
     #[arg(long, conflicts_with = "isolated")]
     pub package: Option<PackageName>,
 
+    /// Specifies the Python script where the dependency will be removed.
+    #[arg(long)]
```

Comment:
I think `--script` should be marked as conflicting with a bunch of other options, like: `--locked`, `--frozen`, `--dev`, etc. Or, we can warn in `add` and `remove` if those are set, to indicate that they're ignored, like in https://github.com/astral-sh/uv/pull/5977. Fine to just follow that pattern.

Reply:
I've added a warning for the other options, let me know if I missed something

Reply:
Looks good. I marked `--dev` and `--optional` as conflicting, since those should truly error. The rest are more "redundant".